ml-conversational-analytic-tool/commentAnalysis.py

```diff
@@ -4,9 +4,9 @@
 import argparse
 
 import nltk
+from nltk.sentiment.vader import SentimentIntensityAnalyzer
```

Comment:
It's generally better to import modules, not classes.
```suggestion
from nltk.sentiment import vader
```